app/models/question_type.rb

```diff
@@ -3,16 +3,22 @@ class QuestionType
   attr_reader :name, :odk_name, :properties
 
   @@attributes = [
-    {:name => "text", :odk_name => "string", :properties => %w(form_printable smsable textual headerable)},
-    {:name => "long_text", :odk_name => "string", :properties => %w(form_printable smsable textual)},
-    {:name => "integer", :odk_name => "int", :properties => %w(form_printable smsable numeric headerable)},
-    {:name => "decimal", :odk_name => "decimal", :properties => %w(form_printable smsable numeric headerable)},
-    {:name => "location", :odk_name => "geopoint", :properties => %w()},
-    {:name => "select_one", :odk_name => "select1", :properties => %w(form_printable has_options smsable headerable)},
-    {:name => "select_multiple", :odk_name => "select", :properties => %w(form_printable has_options smsable headerable)},
-    {:name => "datetime", :odk_name => "dateTime", :properties => %w(form_printable temporal has_timezone smsable headerable)},
-    {:name => "date", :odk_name => "date", :properties => %w(form_printable temporal smsable headerable)},
-    {:name => "time", :odk_name => "time", :properties => %w(form_printable temporal smsable headerable)}
+    { name: 'text', odk_name: 'string', properties: %w(form_printable smsable textual headerable) },
+    { name: 'long_text', odk_name: 'string', properties: %w(form_printable smsable textual) },
+    { name: 'integer', odk_name: 'int', properties: %w(form_printable smsable numeric headerable) },
+    { name: 'decimal', odk_name: 'decimal', properties: %w(form_printable smsable numeric headerable) },
+    { name: 'location', odk_name: 'geopoint', properties: %w() },
+    { name: 'select_one', odk_name: 'select1', properties: %w(form_printable has_options smsable headerable) },
+    { name: 'select_multiple', odk_name: 'select', properties: %w(form_printable has_options smsable headerable) },
+    { name: 'datetime', odk_name: 'dateTime', properties: %w(form_printable temporal has_timezone smsable headerable) },
+    { name: 'date', odk_name: 'date', properties: %w(form_printable temporal smsable headerable) },
+    { name: 'time', odk_name: 'time', properties: %w(form_printable temporal smsable headerable) },
+    { name: 'image', odk_name: 'image/*', properties: %w(multimedia) },
```

Comment:
I had gotten that from https://opendatakit.org/help/form-design/body/ which made no mention of `binary` as a type and I couldn't find any other reference for what the valid types were. Can make that change, but I'm curious where I should have been looking to find that information if not there, for next time.